website/docs/api/misc.mdx

```diff
@@ -0,0 +1,1002 @@
+
+```c
+double cellAreaM2(H3Index h);
+```
+
+</TabItem>
+<TabItem value="python">
+
+```py
+h3.cell_area(h, unit='m^2')
+```
+
+</TabItem>
+<TabItem value="java">
+
+```java
+double cellArea(long h3, AreaUnit unit);
+int cellArea(String h3Address, AreaUnit unit);
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```js
+h3.cellArea(h, h3.UNITS.m2)
+```
+
+```js live
+function example() {
+  const h = '85283473fffffff';
+  return h3.cellArea(h, h3.UNITS.m2);
+}
+```
+
+</TabItem>
+</Tabs>
+
+Exact area of specific cell in square meters.
+
+## cellAreaRads2
+
+<Tabs
+  groupId="language"
+  defaultValue="c"
+  values={[
+    {label: 'C', value: 'c'},
+    {label: 'Python', value: 'python'},
+    {label: 'Java', value: 'java'},
+    {label: 'JavaScript (Live)', value: 'javascript'},
+  ]
+}>
+<TabItem value="c">
+
+```c
+double cellAreaRads2(H3Index h);
+```
+
+</TabItem>
+<TabItem value="python">
+
+```py
+h3.cell_area(h, unit='rads^2')
+```
+
+</TabItem>
+<TabItem value="java">
+
+```java
+double cellArea(long h3, AreaUnit unit);
+int cellArea(String h3Address, AreaUnit unit);
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```js
+h3.cellArea(h, h3.UNITS.rads2)
+```
+
+```js live
+function example() {
+  const h = '85283473fffffff';
+  return h3.cellArea(h, h3.UNITS.rads2);
+}
+```
+
+</TabItem>
+</Tabs>
+
+Exact area of specific cell in square radians.
+
+## edgeLengthKm
+
+<Tabs
+  groupId="language"
+  defaultValue="c"
+  values={[
+    {label: 'C', value: 'c'},
+    {label: 'Python', value: 'python'},
+    {label: 'Java', value: 'java'},
+    {label: 'JavaScript (Live)', value: 'javascript'},
+  ]
+}>
+<TabItem value="c">
+
+```c
+double edgeLengthKm(int res);
+```
+
+</TabItem>
+<TabItem value="python">
+
+```py
+h3.edge_length(res, unit='km')
+```
+
+</TabItem>
+<TabItem value="java">
+
+```java
+double edgeLength(int res, LengthUnit unit);
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```js
+h3.edgeLength(res, h3.UNITS.km)
+```
+
+```js live
+function example() {
+  const res = 5;
+  return h3.edgeLength(res, h3.UNITS.km);
+}
+```
+
+</TabItem>
+</Tabs>
+
+Average hexagon edge length in kilometers at the given resolution.
+
+## edgeLengthM
+
+<Tabs
+  groupId="language"
+  defaultValue="c"
+  values={[
+    {label: 'C', value: 'c'},
+    {label: 'Python', value: 'python'},
+    {label: 'Java', value: 'java'},
+    {label: 'JavaScript (Live)', value: 'javascript'},
+  ]
+}>
+<TabItem value="c">
+
+```c
+double edgeLengthM(int res);
+```
+
+</TabItem>
+<TabItem value="python">
+
+```py
+h3.edge_length(res, unit='m')
+```
+
+</TabItem>
+<TabItem value="java">
+
+```java
+double edgeLength(int res, LengthUnit unit);
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```js
+h3.edgeLength(res, h3.UNITS.km)
+```
+
+```js live
+function example() {
+  const res = 5;
+  return h3.edgeLength(res, h3.UNITS.km);
+}
+```
+
+</TabItem>
+</Tabs>
+
+Average hexagon edge length in meters at the given resolution.
+
+
+## exactEdgeLengthKm
```

Comment:
The three `exactEdgeLength` functions are showing up as `0` on all of the examples. Bug in h3-js, or bug here?

Reply:
Looks like they're all being given a cell index, not an edge index, as input. Try `"115283473fffffff"`